docs/navigation.html

```diff
@@ -33,6 +33,7 @@
   <li {% if page.path == 'useDeleteMany.md' %} class="active" {% endif %}><a class="nav-link" href="./useDeleteMany.html"><code>useDeleteMany</code></a></li>
   <li><a class="nav-link external" href="https://marmelab.com/ra-enterprise/modules/ra-realtime#real-time-hooks.html" target="_blank"><code>useSubscribe</code><img class="premium" src="./img/premium.svg" /></a></li>
   <li {% if page.path == 'useGetTree.md' %} class="active" {% endif %}><a class="nav-link" href="./useGetTree.html"><code>useGetTree</code><img class="premium" src="./img/premium.svg" /></a></li>
+  <li {% if page.path == 'useGetRecordId.md' %} class="active" {% endif %}><a class="nav-link" href="./useGetRecordId.html"><code>useGetRecordId</code></a></li>
```

Comment:
It shouldn't be in this section, but rather in the Hooks section with useRecordContext